ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -1146,6 +1146,21 @@ private boolean isChangeTypeDate(AttributeChange attributeChange, Metacard resul
   }
 
   private List<Result> getMetacardHistory(String id) {
+    Filter resourceFilter = filterBuilder.attribute(Core.ID).is().equalTo().text(id);
+    Metacard resource = null;
+    try {
+      List<Result> results =
+          catalogFramework.query(new QueryRequestImpl(new QueryImpl(resourceFilter))).getResults();
+      if (results.isEmpty()) {
+        LOGGER.trace("Unable to find resource {} inside history endpoint.", id);
+        return new ArrayList<Result>();
```

Comment:
✏️ Alternatively, `Collections.emptyList()`. I guess you could also just return the `results` empty list, but I think the former is a bit clearer.